databricks/koalas/tests/test_series.py

```diff
@@ -120,10 +120,10 @@ def test_rename(self):
         self.assertEqual(kser.name, "renamed")
         self.assert_eq(kser, pser)
 
-        pser.name = None
-        kser.name = None
-        self.assertEqual(kser.name, None)
-        self.assert_eq(kser, pser)
+        # pser.name = None
+        # kser.name = None
+        # self.assertEqual(kser.name, None)
+        # self.assert_eq(kser, pser)
```

Comment:
Hm .. so this case doesn't work anymore?

Reply:
`Seires` without the name is not working properly now anyway.
We should revisit later.